.github/workflows/main.yaml

```diff
@@ -180,7 +183,7 @@ jobs:
       TEST_RESULTS_DIR: ${{ matrix.name }}_test_results
       TEST_ARTIFACTS_KEY: ${{ matrix.platform }}_${{ matrix.name }}_test_artifacts
       IS_POSTSUBMIT: ${{ github.event_name == 'schedule' || github.event_name == 'push' }}
-      TESTING_ENABLED: ${{ needs.initialize.outputs.test_on_host == 'true' || needs.initialize.outputs.test_on_device == 'true' }}
+      TESTING_ENABLED: ${{ needs.initialize.outputs.test_on_host == 'true' || needs.initialize.outputs.test_on_device == true }}
```

Comment:
Did you mean to remove the quotes from just one of the `true`s here?

Reply:
fixed